Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,146 @@
-import React from 'react';
-import './App.css';
-
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
+import React from "react";
+import "./App.css";
+import Display from "./components/DisplayComponents/CalculatorDisplay";
+import WhiteButtons from "./components/ButtonComponents/NumberButton";
+import RedButtons from "./components/ButtonComponents/ActionButton";
```

Comment:
using inconsistent names for your components when importing is really asking for trouble.
this can easily cause confusion even in a relatively small program like this calculator.
best practice is to import the components EXACTLY as the files are named

```
import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
import NumberButton from "./components/ButtonComponents/NumberButton";
import ActionButton from "./components/ButtonComponents/ActionButton";
```